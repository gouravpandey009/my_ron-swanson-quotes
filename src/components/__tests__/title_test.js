import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Title from '../title';

Enzyme.configure({adapter: new Adapter()});

test('title renders correctly', () => {
  const component = shallow(<Title />)
  expect(component).toMatchSnapshot();
})