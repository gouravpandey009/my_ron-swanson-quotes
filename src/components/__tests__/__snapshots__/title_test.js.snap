// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quote table renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Title />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <span
        className="ui header center aligned"
>
        <div>
                <img
                        className="ui centered aligned circular image medium"
                        id="ron_image"
                        src="ronavatar.jpg"
                />
        </div>
        <h2>
                Rate Ron's Quotes
        </h2>
</span>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div>
            <img
                        className="ui centered aligned circular image medium"
                        id="ron_image"
                        src="ronavatar.jpg"
            />
</div>,
          <h2>
            Rate Ron's Quotes
</h2>,
        ],
        "className": "ui header center aligned",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              className="ui centered aligned circular image medium"
              id="ron_image"
              src="ronavatar.jpg"
/>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "ui centered aligned circular image medium",
              "id": "ron_image",
              "src": "ronavatar.jpg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Rate Ron's Quotes",
          },
          "ref": null,
          "rendered": "Rate Ron's Quotes",
          "type": "h2",
        },
      ],
      "type": "span",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <span
          className="ui header center aligned"
>
          <div>
                    <img
                              className="ui centered aligned circular image medium"
                              id="ron_image"
                              src="ronavatar.jpg"
                    />
          </div>
          <h2>
                    Rate Ron's Quotes
          </h2>
</span>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              <img
                            className="ui centered aligned circular image medium"
                            id="ron_image"
                            src="ronavatar.jpg"
              />
</div>,
            <h2>
              Rate Ron's Quotes
</h2>,
          ],
          "className": "ui header center aligned",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                className="ui centered aligned circular image medium"
                id="ron_image"
                src="ronavatar.jpg"
/>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "ui centered aligned circular image medium",
                "id": "ron_image",
                "src": "ronavatar.jpg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Rate Ron's Quotes",
            },
            "ref": null,
            "rendered": "Rate Ron's Quotes",
            "type": "h2",
          },
        ],
        "type": "span",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`title renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Title />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <span
        className="ui header center aligned"
>
        <div>
                <img
                        className="ui centered aligned circular image medium"
                        id="ron_image"
                        src="ronavatar.jpg"
                />
        </div>
        <h2>
                Rate Ron's Quotes
        </h2>
</span>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div>
            <img
                        className="ui centered aligned circular image medium"
                        id="ron_image"
                        src="ronavatar.jpg"
            />
</div>,
          <h2>
            Rate Ron's Quotes
</h2>,
        ],
        "className": "ui header center aligned",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              className="ui centered aligned circular image medium"
              id="ron_image"
              src="ronavatar.jpg"
/>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "ui centered aligned circular image medium",
              "id": "ron_image",
              "src": "ronavatar.jpg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Rate Ron's Quotes",
          },
          "ref": null,
          "rendered": "Rate Ron's Quotes",
          "type": "h2",
        },
      ],
      "type": "span",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <span
          className="ui header center aligned"
>
          <div>
                    <img
                              className="ui centered aligned circular image medium"
                              id="ron_image"
                              src="ronavatar.jpg"
                    />
          </div>
          <h2>
                    Rate Ron's Quotes
          </h2>
</span>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              <img
                            className="ui centered aligned circular image medium"
                            id="ron_image"
                            src="ronavatar.jpg"
              />
</div>,
            <h2>
              Rate Ron's Quotes
</h2>,
          ],
          "className": "ui header center aligned",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                className="ui centered aligned circular image medium"
                id="ron_image"
                src="ronavatar.jpg"
/>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "ui centered aligned circular image medium",
                "id": "ron_image",
                "src": "ronavatar.jpg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Rate Ron's Quotes",
            },
            "ref": null,
            "rendered": "Rate Ron's Quotes",
            "type": "h2",
          },
        ],
        "type": "span",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
